chem/module_optical_averaging.F

```diff
@@ -3716,15 +3719,17 @@ subroutine optical_prep_gocart(nbin_o, chem, alt,relhum,          &
 !      WRITE(*,*)'         ','       ',(dlo_sectm(n),n=1,nbin_o)
 !      WRITE(*,*)'         ','       ',(dhi_sectm(n),n=1,nbin_o)
         dustfrc_goc8bin=0.
-        dlogoc=0.46*2.E-6 ! Begin lower dust bin, makes upper limit diam 20 micron diameter 
+        !dlogoc=0.46*2.E-6 ! Begin lower dust bin, makes upper limit diam 20 micron diameter 
        do m =1, ndust  ! loop over dust size bins
-       dhigoc = 2.*2.*reff_dust(m)-dlogoc ! hi diameter limit (m)
+       !dhigoc = 2.*2.*reff_dust(m)-dlogoc ! hi diameter limit (m)  ! A. Ukhov
```

Comment:
@saneku 
Alexander,
Another example of maybe just removing the the line

Reply:
done